### PR TITLE
Extensions: Add WooCommerce base extension

### PR DESCRIPTION
**Pull request by coderkevin**
This adds the base extension for WooCommerce.
It's only for dev builds and it's just a placeholder page for now.

### CODE REVIEW

File: client/extensions/woocommerce/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			<Card>
+				<p>This is the starting point of something great!</p>
+				<p>This will be the home for your WooCommerce integration with WordPress.com.</p>
+			</Card>
+		</Main>
+	), document.getElementById( 'primary' ), context.store );
+};
+
+export default function() {
+	page( '/woocommerce/:site?', siteSelection, navigation, render );
```
Comment (mtias) [inline client/extensions/woocommerce/index.js]: Is this the URL we'd want to use? Could it perhaps be `/store`?

Comment (mtias) [inline client/extensions/woocommerce/index.js]: cc @kellychoffman

**Comment by mtias** [comment]
Looks good!

File: client/extensions/woocommerce/index.js (hunk 1 above)
Comment (coderkevin) [inline client/extensions/woocommerce/index.js]: If this were only for WordPress.com sites, I think `/store` would be best. However, for Jetpack connected sites, does it make sense?

Comment (kellychoffman) [inline client/extensions/woocommerce/index.js]: +1 for /store. It also matches the navigation item label.

Comment (toddwilkens) [inline client/extensions/woocommerce/index.js]: +1 for /store.

Comment (coderkevin) [inline client/extensions/woocommerce/index.js]: `/store` it is!

**Comment by coderkevin** [comment]
So, the URL is now `/store`, but are we comfortable calling the extension/module `woocommerce` still?

File: client/extensions/woocommerce/index.js (hunk 1 above)
Comment (mtias) [inline client/extensions/woocommerce/index.js]: I think so, should be consistent.

**Comment by mtias** [comment]
That seems fine to me. I'd still defer to @kellychoffman or @toddwilkens on the URL.

**Comment by coderkevin** [comment]
Sounds fine to me too. If we feel strongly enough in the future, we can always change it.

File: client/extensions/woocommerce/index.js (hunk 1 above)
Comment (mtias) [inline client/extensions/woocommerce/index.js]: @coderkevin did you mean to change it on this PR? Looks like it's still woocommerce :)

**Pull request merged**